rllib/examples/env/deterministic_envs.py

```diff
@@ -0,0 +1,29 @@
+import gym
+
+
+class DeterministicCartPole(gym.Env):
+    def __init__(self, seed=0):
+        self.env = gym.make("CartPole-v0")
```

Comment:
not sure if we needed to add these as specific environments. Could we remove these and then just set seed directly. 

technically thats what we do right now. 


if you pass seed and the environment has a seed function attribute then we call it to fix the seed.

Reply:
still keeping the deterministic_env module since it allows us to share the env boiler plate code across all the tests, but made them using the existing seedable gym classes instead of creating separate wrappers that needed to be maintained.